JetNews/app/src/main/java/com/example/jetnews/ui/article/ArticleScreen.kt

```diff
@@ -72,18 +73,18 @@ fun ArticleScreen(postId: String) {
                 title = {
                     Text(
                         text = "Published in: ${post.publication?.name}",
-                        style = MaterialTheme.typography().subtitle2
+                        style = MaterialTheme.typography.subtitle2.copy(color = contentColor())
```

Comment:
Should we link to the bug every time we type `.copy(color = contentColor()`?

Reply:
Adding.

Reply:
Just this one place that I see.